src/tools/wdump/wdump.cpp

```diff
@@ -4126,7 +4126,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
     MSG msg = {};
 
     while (WM_QUIT != msg.message) {
-        if (PeekMessage(&msg, nullptr, 0, 0, PM_REMOVE)) {
+        if (GetMessage(&msg, nullptr, 0, 0)) {
```

Comment:
Perhaps add a comment so that future engineer does not change it back to `PeekMessage`, because it mismatches decomp.